.vsts-ci/releaseStage.yml

```diff
@@ -0,0 +1,43 @@
+parameters:
+  channel: 'preview'
+  vmImage: 'ubuntu-16.04'
+  acr: 'All'
+  useacr: 'false'
+  osFilter: 'All'
+
+stages:
+  - stage: GenerateYaml_${{ parameters.channel }}
+    dependsOn: []
+    displayName: Build ${{ parameters.channel }}
+    jobs:
+    - job: GenerateYaml_${{ parameters.channel }}
+      pool:
+        vmImage: ubuntu-16.04
+      displayName: Generate Matrix YAML ${{ parameters.channel }}
+      steps:
+      - pwsh: |
+          ./build.ps1 -GenerateMatrixJson -Channel ${{ parameters.channel }} -Verbose -Acr ${{ parameters.acr }} -OsFilter  ${{ parameters.osFilter }}
+        displayName: Generate Matrix YAML
+        condition: succeededOrFailed()
+        name: matrix
+      - pwsh: |
+          dir env:matrix_* | out-string -Width 1000
```

Comment:
maybe also convertfrom-json to make it easier to review

Reply:
The json is actually easier to read... When we get something like this

```

Name : json
json : @{a=}
```